samples/androidNativeActivity/README.md

```diff
@@ -0,0 +1,8 @@
+# Android Native Activity
+ This example shows how to build an Android Native Activity.
+
+The example will render a textured dodecahedron using OpenGL ES library. You can rotate it with your fingers.
```

Comment:
It can be rotated with fingers. 
Please make sure that Android SDK version 25 is installed, using Android SDK manager.